docs/community/configuration.rst

```diff
@@ -0,0 +1,102 @@
+Configuration for compiling GluonNLP website
+-----------------------------------------------------------------
+
+ | The GluonNLP docs website is at https://gluon-nlp.mxnet.io (release branch), or https://gluon-nlp.mxnet.io/master/index.html (master branch). Its source code is at https://github.com/dmlc/gluon-nlp .
+
+Currently the GluonNLP website is constructed from the source code via CI automatically. Nevertheless, if a maintainer wants to make some modification and preview its changes, it will become difficult. I am writing this documentation to note down how to compile the entire website locally for future references.
+
+Website Structure
+~~~~~~~~~~~~~~~~~
+
+Currently the docs part contain four sections: Model Zoo, Examples, API and Community. It should be noted that the model zoo is a link redirecting to the ``scripts`` folder in the parent folder. The other three folders are used exclusively by the docs website. Also, another point that should be paid attention to is that the three different sections use different file format system for compiling - they are inconsistent. So when you work on different sections of the docs website, you should  pay attention to handle the different sections with care.
```

Comment:
"Also, another point that should be paid attention to is that the three different ..." ->
simplify it to "Also, three different ..."